frontend/src/TakerApp.tsx

```diff
@@ -123,12 +125,9 @@ export default function App() {
                 <Grid templateColumns="repeat(6, 1fr)" gap={4}>
                     <GridItem colStart={1} colSpan={2}>
                         <Wallet walletInfo={walletInfo} />
+                        {/*TODO: Do we need this? does it make sense to only display the price from the order?*/}
```

Comment:
Is this still up2date?

Reply:
removed, as we're likely to have the price feed.